packages/phoenix-ng/projects/phoenix-ui-components/package.json

```diff
@@ -13,7 +13,8 @@
     "dist"
   ],
   "scripts": {
-    "prepublishOnly": "ng build phoenix-ui-components --prod"
+    "prepublishOnly": "node --eval \"console.error('ERROR: Trying to publish a package that has been compiled by NGCC. This is not allowed.\\nPlease delete and rebuild the package, without compiling with NGCC, before attempting to publish.\\nNote that NGCC may have been run by importing this package into another project that is being built with Ivy enabled.\\n')\" && exit 1",
```

Comment:
Ok, so this is what the ngcc compiler does. Please discard all changes in this file and manually update the `three` peer dependency to `0.122.0`. Otherwise, this will cause problems during publishing.

Reply:
I force pushed a fix...

Reply:
Thanks! (And sorry I missed this!)